server/models/blog.js

```diff
@@ -0,0 +1,40 @@
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+//var ObjectId = Schema.Types.objectId;
```

Comment:
The posts and comments will be refactored to take account for object ids rather than strings for authors. However, I'm not certain how to make sure we're pulling the names for these properties from the other collections rather than the IDs. Once I figure that out, we'll refactor this model.